### ISSUE TITLE
dependency cycle with py-evm backend

### ISSUE THREAD

**Issue by palango** [ethereum/eth-tester#81]
* Version: 0.1.0b23
* Python: 3.6
* OS: osx

### What was wrong?

I specify `eth-tester[py-evm]>=0.1.0-beta.23,<1.0` in my dependencies. `pip install -r requirements.txt` works fine with that.

However, when trying to run `python setup.py develop` on my package I get the following error:

```
error: eth-tester 0.1.0b23 is installed but eth-tester==0.1.0b21 is required by {'py-evm'}
```

eth-tester requires py-evm 0.2.0a14, but this one in return requires eth-tester 0.1.0b21: https://github.com/ethereum/py-evm/blob/cc877851d4e110eabfba1bfe544bfe187b51ebea/setup.py#L29

### How can it be fixed?

Fix the dependency cycle.

**Comment by pipermerriam** [comment]
I'll get a new release of these out today with the dependency cycle resolved.

**Comment by palango** [comment]
Any progress on this front?

**Comment by pipermerriam** [comment]
@palango yes but not good news.  Working on cutting these releases but it's requiring some updates to eth-tester to support the upstream changes to py-evm.

**Comment by pipermerriam** [comment]
Should be fixed with `0.1.0-beta.24`
